packages/components/atoms/f-toggle/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "@justeat/f-toggle",
+  "description": "Fozzie Toggle - A component to toggle a single setting on/off",
+  "version": "0.1.0",
+  "main": "dist/f-toggle.umd.min.js",
+  "maxBundleSize": "5kB",
+  "files": [
+    "dist",
+    "test-utils"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://github.com/justeat/fozzie-components/tree/master/packages/components/atoms/f-toggle",
+  "contributors": [
+    "Github contributors <https://github.com/justeat/fozzie-components/graphs/contributors>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:justeat/fozzie-components.git"
+  },
+  "bugs": {
+    "url": "https://github.com/justeat/fozzie-components/issues"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": "^12 || ^14 || ^16"
```

I'd probably just include 14 and 16 here now (and potentially include 18 as well)

```suggestion
    "node": "^14 || ^16 || ^18"
```